pages/record/_.vue

```diff
@@ -9,31 +9,53 @@
     data-qa="record page"
     class="mt-5"
   >
-    <b-row class="mb-3 mx-0 card p-3">
+    <b-row class="mb-3">
       <b-col
         cols="12"
-        class="mb-3 px-0"
+        lg="9"
       >
-        <MediaPresentation
-          :codec-name="edmIsShownBy.edmCodecName"
-          :image-link="image.link"
-          :image-src="image.src"
-          :mime-type="edmIsShownBy.ebucoreHasMimeType"
-          :url="edmIsShownBy.about"
-          :width="edmIsShownBy.ebucoreWidth"
-          :height="edmIsShownBy.ebucoreHeight"
-        />
+        <div class="card p-3">
+          <div
+            class="card-grid"
+            :class="isRichMedia && 'card-grid-richmedia'"
+          >
+            <h1
```

Comment:
As this may be in a language other than the user is viewing the UI, we need to include the `lang` attribute here.

Given that we will need to do this on every metadata field, or at least those not in the same language as the UI, I wonder if we should have a new component to handle this, or refactor components/record/LangMap.vue for this purpose.

Reply:
I have made this part of the filter. The filter returns the text output and the lang code that can be used on the HTML element. 

The language of the document `html lang="en"` was always English, even if you switched languages, so I made a fix for that too.